### DISCUSSION TITLE
Translation problem

### DISCUSSION THREAD

**Discussion by junhanzang** [openai/whisper#1049] [General]
When i try translation English to other words, French is working but Japanese, Korean doesn't work.
My try code is 
!whisper audio.mp3 --model medium --task transcribe --language Korean
either 
!whisper audio.mp3 --model medium --task translate --language Korean

It is only possible to translate other word to english? Then, why French is work?

**Comment by jongwook** [comment]
The model was not trained for translations to languages other than English, and translation to French as you saw is quite coincidental due to the weakly supervised nature of the training process. See also:

- #110 
- #649

**Reply by junhanzang** [reply]
Thank you for your kind response.
